<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BareMetal::BareMetalDeviceConfigurationWidget</class>
 <widget class="QWidget" name="BareMetal::BareMetalDeviceConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="gdbHostLabel">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>GDB host:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="gdbHostLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="gdbPortLabel">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>GDB port:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="gdbPortSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>3333</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="gdbCommandsLabel">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>GDB commands:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="gdbCommandsTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
